devel/xcode-build-server/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        SolaWing xcode-build-server 1.0.0 v
+version             1.0.0
+revision            0
+platforms           {darwin}
+license             MIT
```

Comment:
"darwin" is the default but if this is the "usual" Python it could be "{darwin any} if it installs the same files on each OS version. Otherwise, then line can be removed.

Reply:
Removed.

Reply:
if it only install Python files, then it should become "platforms {darwin any}

Reply:
The project is only useful on a platform where Xcode is available, so `any` seems inappropriate.

Reply:
no, it isn't `{darwin any}` just means that the installed files will be the same on all macOS versions, which is the case here. See the description in the guide - [here](https://guide.macports.org/index.html#reference.keywords).

Reply:
I see, I misunderstood the syntax. So this should probably be `{macosx any}` then, since I don't think Xcode is on plain Darwin.